internal/log/hook_filename.go

```diff
@@ -64,7 +64,7 @@ func (f *FileNameHook) findCaller() (file, function string, line int) {
 	const maxFrames = 10
 	for i := 0; i < maxFrames; i++ {
 		// The amount of frames to be skipped to land at the actual caller
-		const skipFrames = 5
+		const skipFrames = 6
```

Comment:
why'd we bump this?

Reply:
Because we now add 1 function depth on top of the stack with the new intermediate types and calling into the server package. Otherwise the file name would point to the wrong function (always to the grpc middleware).

Reply:
ah yes I see, thanks